opal/util/proc.c

```diff
@@ -215,14 +198,36 @@ char *opal_get_proc_hostname(const opal_proc_t *proc)
     if (proc == opal_proc_my_name) {
         return strdup(opal_process_info.nodename);
     }
+    return strdup("unknown");
+#if 0
```

Comment:
We shouldn't #if 0 code, but just remove it.  But we should really add a comment about what this function does in the new scope (and that it's replaced when using the full library).  But bigger picture, why is this code even in the util library?